src/main/java/XlsExcelFile.java

```diff
@@ -62,7 +63,7 @@ public XlsExcelFile(String filePath) {
      * @param columnIdx cell column index
      */
     @Override
-    public void writeCell(String content, int rowIdx, int columnIdx) {
+    public void writeCell(CellContent content, int rowIdx, int columnIdx) {
```

Comment:
With your patch we could define the behaviour of the method `writeCell` in the abstract class `Excel`. This would prevent us from the existing code duplication in `XlsExcelFile` and `XlsxExcelFile`.